client-docs/README.md

```diff
@@ -0,0 +1,377 @@
+# Mapeo Core Client API
+
+## Table of Contents
+
+- [Top Level Client](#top-level-client)
+
+  - [Types](#types)
+  - [`client.createProject`](#clientcreateproject)
+  - [`client.getProject`](#clientgetproject)
+  - [`client.listProjects`](#clientlistprojects)
+  - [`client.invite`](#clientinvite)
+    - [`invite.accept`](#inviteaccept)
+    - [`invite.reject`](#invitereject)
+    - Events
+      - [`'invite-received'`](#invite-received)
+
+- [Project Client](#project-client)
+  - [Types](#types-1)
+  - [`project.$getProjectSettings`](#projectgetprojectsettings)
+  - [`project.$updateProjectSettings`](#projectupdateprojectsettings)
+  - [`project.$leave`](#projectleave)
+  - [`project.$sync`](#projectsync)
+    - [`$sync.getState`](#syncgetstate)
+    - [`$sync.setDiscovery`](#syncsetdiscovery)
+    - [`$sync.setSync`](#syncsetsync)
+    - [`$sync.stop`](#syncstop)
+    - Events
+      - [`'sync-state'`](#sync-state)
+  - [`project.$member`](#projectmember)
+    - [`$member.invite`](#memberinvite)
+    - [`$member.getById`](#membergetbyid)
+    - [`$member.getMany`](#membergetmany)
+    - [`$member.update`](#memberupdate)
+    - [`$member.remove`](#memberremove)
+    - Events
+      - [`'member-update'`](#member-update)
+  - [`project.$blob`](#projectblob)
+    - [`$blob.getUrl`](#blobgeturl)
+    - [`$blob.create`](#blobcreate)
+  - [DataType](#datatype)
+    - [Types](#types-2)
+    - [`dataType.create`](#datatypecreate)
+    - [`dataType.getByDocId`](#datatypegetbydocid)
+    - [`dataType.getByVersionId`](#datatypegetbyversionid)
+    - [`dataType.getMany`](#datatypegetmany)
+    - [`dataType.update`](#datatypeupdate)
+    - [`dataType.delete`](#datatypedelete)
+
+## Top Level Client
+
+Client interface for managing projects and invites. Provides an event emitter-like interface so it can emit and subscribe to events.
+
+### Types
+
+```ts
+import { Opaque } from "type-fest";
+
+type ProjectId = Opaque<string>;
```

Comment:
All `Opaque<string>` can be assigned to each other, they need a token to get the benefits of Opaque types. However I've just [added these to mapeo-schema](https://github.com/digidem/mapeo-schema/pull/78) so you would import from there, e.g.

```
import { ProjectDocId } from '@mapeo/schema'

type ProjectInfo = {
  docId: ProjectDocId
}
```

Reply:
Is this still relevant? looks like you decided against adding them